guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: single
+title:  "Add a Canary stage to a pipeline"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+
+If you have [enabled canary for your application](/setup/canary/) and have one
+or more [canary configs prepared](), you can now add a canary stage to your
+pipeline and configure it to perform canary analysis for your deployment,
+against the target environment.
+
+This stage type is for the canary _analysis_. The canary _deployment_ is specified
+upstream from this stage.
+
+## About the canary stage
+
+The canary analysis can be performed over data points collected beginning from
+the moment of execution and into the future, or it can be performed over a
+specified time interval.
+
+### Real-time versus retrospective analysis
+
+A real-time analysis means that the canary analysis is performed over a time
+interval beginning at the moment of execution. In a retrospective analysis the
+canary analysis is performed over an explicitly specified time interval (likely
+in the past).
+
+### Metric scope
+
+Metric scope is the where and when for your canary analysis. It describes the
+specific baseline and canary server groups, the start and end times and interval,
+and the cloud resource on which the baseline and canary are running.
+
+You can also
+
+
+
+
+## Configure the canary stage
+
+<!-- something here about where in the pipeline you'd put the canary stage, and
+other permutations, like multiple stages? -->
+
+1. In the pipeline in which you will run the canary, click __Add stage__.
+
+   This pipeline needs to be in an application that has access to the [canary
+   configuration](/guides/user/canary/config/) you want to use.
+
+1. For __Type__ select __Canary__.
+
+1. Give the stage a name, and use the __Depends On__ field to position the stage
+downstream of its dependencies.
+
+   ![Canary stage declaration](/guides/user/canary/stage/canary_stage_top.png)
+
+1. Select the __Analysis Type__&mdash;either __Real Time__ or __Retrospective__.
+
+   * __Real Time__
+
+     The analysis happens for a specified time period, beginning when the stage
+     executes.
+
+     For __Real Time__, also specify the number of hours to run (__Lifetime__).
+
+   * __Retrospective__
+
+     Analysis occurs over some specified period. Typically, this is done
+     for a time period in the past, against a baseline deployment and a canary
+     deployment which have already been running before this canary analysis
```

Comment:
May even refer to resources which no longer exist, but for which there are still published time series data.